bool doIt
---
string status_message
